project_description/__init__.py

```diff
@@ -0,0 +1,4 @@
+# © 2015 Antiun Ingeniería S.L. - Jairo Llopis
```

Comment:
copyright notice not necessary in `__init__.py`

Reply:
@elicoidal and @pedrobaeza Should I remove this also in models/init.py?

Reply:
Yeah, I think so. `__init__.py` files are not code itself, just a tool for loading the actual code.